### PR TITLE
144 general refactoring

### PR DESCRIPTION
**Pull request by manuelkoso**
Just small refactoring following the SonarQube analysis. I'll create issues for bigger refactoring.

### CODE REVIEW

File: core/src/it/units/sdm/project/game/gui/screens/GameScreen.java [hunk 1]
```diff
@@ -36,7 +37,7 @@ public GameScreen(@NotNull FreedomGame game) {
         stage = new Stage(new FitViewport(GAME_SCREEN_WORLD_WIDTH, GAME_SCREEN_WORLD_HEIGHT), new SpriteBatch());
         VisTable container = new VisTable();
         logArea = new VisTextArea("Welcome to Freedom ");
-        logArea.setPrefRows(game.getNumberOfRowsAndColumns() * 2);
+        logArea.setPrefRows(game.getNumberOfRowsAndColumns() * 2.0f);
```
Comment (peiva-git) [inline core/src/it/units/sdm/project/game/gui/screens/GameScreen.java:40]: Is this a SonarQube suggestion? Also, 2f might be sufficient

Comment (manuelkoso) [reply core/src/it/units/sdm/project/game/gui/screens/GameScreen.java:40]: yes, you are right

Comment (peiva-git) [reply core/src/it/units/sdm/project/game/gui/screens/GameScreen.java]: I reverted back to the version with no casting then

Comment (manuelkoso) [reply core/src/it/units/sdm/project/game/gui/screens/GameScreen.java]: I have already refactored that

**Review by peiva-git — APPROVED** [review]
Good!

**Pull request merged**